client/transactions/list/test/index.js

```diff
@@ -293,4 +293,14 @@ describe( 'Transactions list', () => {
 		const { container } = render( <TransactionsList /> );
 		expect( container ).toMatchSnapshot();
 	} );
+
+	describe( 'test getColumns', () => {
+		test( 'should return base column structure', () => {
+			expect( getColumns() ).toMatchSnapshot();
+		} );
+
+		test( 'should include subscriptions and deposit columns when both includeDeposit, includeSubscription as passed as true', () => {
+			expect( getColumns( true, true ) ).toMatchSnapshot();
+		} );
+	} );
```

Comment:
Testing this as function doesn't make sense as well as exporting `getColumns` only for testing purposes. Since it's internal method of the component which is rulled by [these lines](https://github.com/Automattic/woocommerce-payments/blob/19e597355b567df4ae0f58b37aec547f38f7f186/client/transactions/list/index.js#L150-L154) we should render the whole list with required state, unless already tested.

Reply:
I give it a second thought and you are right, there is not much use in exporting and testing an internal function, rather what I did now is to test the actual behavior of the download CSV function by simulating a click event. Commit: https://github.com/Automattic/woocommerce-payments/pull/1776/commits/10f4a6afe92276ea1c89fd55758b5ba90af50358

Reply:
Thanks, for this, nice catch!